tailwind.config.js

```diff
@@ -1,6 +1,48 @@
 module.exports = {
   content: ['./src/**/*.{html,js,tsx,ts}'],
-  theme: {},
+  theme: {
+    fontFamily: {
+      sans: ['Nunito', 'sans-serif'],
+      serif: ['Merriweather', 'serif'],
+    },
+    fontSize: {
+      '4xl': [
+        '3rem',
+        {
+          lineHeight: '1.4',
+        },
+      ],
+      '3xl': [
+        '2.25rem',
+        {
+          lineHeight: '1.4',
+        },
+      ],
+      '2xl': [
+        '1.75rem',
+        {
+          lineHeight: '1.2',
+        },
+      ],
+      l: [
+        '1.25rem',
+        {
+          lineHeight: '1.2',
+        },
+      ],
+      s: '1rem',
```

Comment:
I think tailwind's convention is to name this size `sm` instead of `s`

Reply:
done!

Reply:
I keep seeing this line unchanged